src/foundry/common/abstract/document.d.mts

```diff
@@ -2106,6 +2134,22 @@ declare namespace Document {
     _PossibleSubtypesContext<DocumentName> &
     ParentContext<Parent>;
 
+  type CreateDialogOptions<
```

Comment:
If you're going to rename `CreateDialogContext` to `CreateDialogOptions` please make sure to leave around the old name deprecated.

Reply:
hmm, I guess leave it as is? Well do deprecate it, just don't change it as an alias.